HospitalWebApp/Controllers/FeedbackController.cs

```diff
@@ -52,11 +70,36 @@ public IActionResult Create(UserFeedbackDto userFeedbackDto)
 
             return Ok();
         }
+
+        /// <summary>
+        /// Publishes a UserFeedback with a given id.
+        /// </summary>
+        /// <param name="id"></param>
+        /// <returns>An <see cref="IActionResult"/> representing a result of the operation.</returns>
+        [Route("publish/{id}")]
+        [HttpGet]
+        public IActionResult Publish(int id)
+        {
+            //TODO: Check if current user is admin
+            if(_userFeedbackService.Publish(id))
+            {
+                return Ok("Feedback successfully published.");
+            }
+            else
+            {
+                return BadRequest();
+            }
+        }
+
+        private static void UpdateUserFeedbackDate(UserFeedback userFeedback)
```

Comment:
This method was removed before and date is set in userFeedbackDtoToUserFeedback method.

Reply:
Must have missed it while fixing merge conflicts. Removed.